packages/dendron-cli/.vscode/tasks.json

```diff
@@ -67,6 +67,15 @@
         "cwd": "D:/dendron_setup/dendron/"
       },
       "problemMatcher": []
+    },
+    {
+      "label": "publish-pod",
+      "type": "shell",
+      "command": "node --inspect dendron/packages/dendron-cli/lib/bin/dendron-cli publishPod --wsRoot ${workspaceFolder}/../../test-workspace --podId dendron.markdown --vault vault --query root --podSource builtin",
+      "options": {
+        "cwd": "E:/dendron-code/"
```

Comment:
since this is personal to your workspace, can we not check this in to the global task.json? can we get to this path using builtin variables like `${workspaceFolder}` instead? and if so, can you add this to `dev.process` so future folks can also do it?

Reply:
https://github.com/dendronhq/dendron-docs/pull/17

Reply:
not related to this PR, but I see a few `/Users/kevinlin/ ...` directories in `tasks.json` 😄 maybe we should clean that up in a separate PR as well.

Reply:
addressed in the same PR : [f5d96d6](https://github.com/dendronhq/dendron/pull/2352/commits/f5d96d6fba56f55ea0b78fe468db14226961893d).